testing/test_skipping.py

```diff
@@ -382,7 +382,82 @@ def test_func():
         ])
 
 
-class TestSkipif:
+class TestSkip(object):
+    def test_skip_class(self, testdir):
+        testdir.makepyfile("""
+            import pytest
+            @pytest.mark.skip
+            class TestSomething(object):
+                def test_foo(self):
+                    pass
+                def test_bar(self):
+                    pass
+
+            def test_baz():
+                pass
+        """)
+        rec = testdir.inline_run()
+        rec.assertoutcome(skipped=2, passed=1)
+
+    def test_skips_on_false_string(self, testdir):
+        testdir.makepyfile("""
+            import pytest
+            @pytest.mark.skip('False')
```

Comment:
what would you expect this arg to be? Would this be assumed to be a reason? Should it be ignored? I guess reason makes the most sense...